lib/commands/doctor/checks/install-folder-permissions.js

```diff
@@ -16,7 +16,9 @@ function installFolderPermissions(ctx) {
             task: taskTitle
         }));
     }).then(() => {
-        if (ctx.local || !ctx.system.platform.linux || (ctx.argv && ctx.argv['setup-linux-user'] === false)) {
+        const isLocal = ctx.local || ctx.instance && ctx.instance.process.name === 'local';
```

Comment:
actually wait - for clarity it might be good to add a parentheses around the two `&&` conditions

Reply:
Good catch!!!